src/map/script.cpp

```diff
@@ -26122,7 +26178,8 @@ struct script_function buildin_func[] = {
 	BUILDIN_DEF(camerainfo,"iii?"),
 
 	BUILDIN_DEF(achievement_condition,"i"),
-	BUILDIN_DEF(getvariableofinstance,"ri"),
+	BUILDIN_DEF(getinstancevar,"ri"),
+	BUILDIN_DEF2(getinstancevar, "getvariableofinstance","ri"),
```

Comment:
```suggestion
	BUILDIN_DEF2_DEPRECATED(getinstancevar, "getvariableofinstance","ri", "2021-12-05"),
```
Adjust to the date when you merge it. :-)